docs/eigenda/operator-guides/protocol-SLA/protocol-SLA.md

```diff
@@ -4,73 +4,51 @@ title: Protocol SLA
-    1. The blob reaches the end of life `100,800 blocks` after onchain
-    confirmation (roughly `14 days`).
+    1. The blob reaches the end of life `100,800 blocks` after onchain confirmation (roughly `14 days`).
 3. **Serve the blobs it stored**
-   1. Note: strictly, it's just attesting (dispersal) and serving (retrieval),
-   as storing the data is implied by serving (the serving needs to be backed by
-   data stored); separating storing out is to make it more clear here.
+   1. Note: strictly, it's just attesting (dispersal) and serving (retrieval), as storing the data is implied by serving (the serving needs to be backed by data stored); separating storing out is to make it more clear here.
+
+Note: When the operator opts in multiple quorums, the above will apply to each quorum.
 
-Note: When the operator opts in multiple quorums, the above will apply to each
-quorum.
+### Responsibility Lifecycle
 
-These responsibilities are mapped into following stages of \<operator, quorum>'s
-lifecycle:
+These responsibilities are mapped into following stages of \<operator, quorum>'s lifecycle:
 
-- **Live:** from \<operator, quorum>'s registration to deregistration (from block `A`
-to `B-1`)
-  - Note: the \<operator, quorum> will not be requested for dispersal with block
-  `B` as reference block, because the \<operator, quorum> won't be in the state
-  produced by that block.
+- **Live:** from \<operator, quorum>'s registration to deregistration (from block `A` to `B-1`)
+  - Note: the \<operator, quorum> will not be requested for dispersal with block `B` as reference block, because the \<operator, quorum> won't be in the state produced by that block.
 - **Full responsibility:** `attest+store+serve` (until block C)
-  - Note: after \<operator, quorum> opted out, it's still responsible for
-  dispersal for `BLOCK_STALE_MEASURE` blocks. This is because the dispersal can
-  use a reference block that is in the past (but within a `BLOCK_STALE_MEASURE`
-  window).
+  - Note: after \<operator, quorum> opted out, it's still responsible for dispersal for `BLOCK_STALE_MEASURE` blocks. This is because the dispersal can use a reference block that is in the past (but within a `BLOCK_STALE_MEASURE` window).
 - **Partial responsibility (lame duck period):** `store+serve` (until block D)
-  - The operator will continue to be responsible for storing and serving the
-  data it signed until all the data is expired.
+  - The operator will continue to be responsible for storing and serving the data it signed until all the data is expired.
 - **Free:** The operator becomes free of responsibilities starting block `D+1`.
 
-Note: if the operator re-opts in the quorum at any point from `B` to `D`, the above
-lifecycle will be restarted.
+Note: if the operator re-opts in the quorum at any point from `B` to `D`, the above lifecycle will be restarted.
 
-## Accountability
+## Accountability Measures, Policies, and Actions
 
 ### Liveness
 
```

Comment:
Inside the table below, is there a way to break the line between Soft measure and Hard measure?


![Screenshot 2024-04-08 at 12 31 36 PM](https://github.com/Layr-Labs/eigenlayer-docs/assets/99709935/19da7dc6-7c2e-43a2-b6da-49244b0d36c4)

Reply:
Yes - stackoverflow says <br> will fix it. Updated the table to include a BR, please see the revision and let me know if you have concerns.